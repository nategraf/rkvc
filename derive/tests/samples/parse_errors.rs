use rkvc_derive::Attributes;

#[derive(Attributes)]
#[rkvc]
struct A {
    a: u64,
}

#[derive(Attributes)]
#[rkvc(foo = "bar")]
struct B {
    b: u64,
}

#[derive(Attributes)]
#[rkvc(crate_path = rkvc)]
struct C {
    c: u64,
}

fn main() {}
