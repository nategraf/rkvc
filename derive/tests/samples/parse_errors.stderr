error: expected attribute arguments in parentheses: `rkvc(...)`
 --> tests/samples/parse_errors.rs:4:3
  |
4 | #[rkvc]
  |   ^^^^

error: Unknown attribute name
  --> tests/samples/parse_errors.rs:10:8
   |
10 | #[rkvc(foo = "bar")]
   |        ^^^

error: expected string literal
  --> tests/samples/parse_errors.rs:16:21
   |
16 | #[rkvc(crate_path = rkvc)]
   |                     ^^^^
